dds_cli/user.py

```diff
@@ -112,13 +111,64 @@ def __authenticate_user(self):
         except simplejson.JSONDecodeError as err:
             raise dds_cli.exceptions.ApiResponseError(message=str(err))
 
-        # Raise exceptions to log info if not ok response
         if not response.ok:
-            message = response_json.get("message", "Unexpected error!")
             if response.status_code == 401:
-                raise exceptions.AuthenticationError(message=message)
+                raise exceptions.AuthenticationError(
+                    "Authentication failed, incorrect username and/or password."
+                )
+            else:
+                raise dds_cli.exceptions.ApiResponseError(
+                    message=f"API returned an error: {response_json['message']}"
```

Comment:
I'd change to `response_json.get('message', "Unknown Error!")`

Reply:
Agreed, fixed!